web/app/scripts/filterbar/numeric-range-directive.js

```diff
@@ -0,0 +1,66 @@
+(function () {
+    'use strict';
+
+    /* ngInject */
+    function NumericRangeField() {
+        var module = {
+            restrict: 'A',
+            require: '^driver-filterbar',
+            templateUrl: 'scripts/filterbar/numeric-range.html',
+            scope: {
+                data: '=',
+                label: '='
+            },
+            link: function(scope, elem, attrs, filterCtl) {
+                init();
+
+                function init() {
+                    scope.filter = {};
+                    scope.error = {};
+                }
+
+                /**
+                 * A simple wrapper around driver-filterbar's updateFilter function;
+                 *  filters should only be updated when data validates
+                 *
+                 * @param filterLabel {string} label of which field to filter
+                 * @param filterObj {object} filter data
+                 */
+                scope.updateFilter = function(fLabel, fObj) {
+                    if (scope.isMinMaxValid()) {
+                        filterCtl.updateFilter(fLabel, fObj);
+                    }
+                };
+
+                $('div.dropdown.numrange ul.dropdown-menu').on('click', function (event) {
```

Comment:
I had no idea about the '$event' object that angular creates for ng-click - ng-click is way better than setting up (potentially redundant) listeners